app.js

```diff
@@ -6,10 +6,57 @@ app.use(express.json());
 const people = [];
 
 
+app.post("/api/v1/people", (req, res) => {
+  if (!req.body.name) {
+      res
+        .status(400)
+        .json({ error: "Please enter a name" });
```

Comment:
I think this message needs to have a period for the first test to pass.

Reply:
Good day. Please check the push request with corrected codes